frontend/src/components/opportunity/OpportunityDocuments.tsx

```diff
@@ -0,0 +1,76 @@
+import { useTranslations } from "next-intl";
+import { Link, Table } from "@trussworks/react-uswds";
+
+interface OpportunityDocument {
+  opportunity_attachment_type: string;
+  file_name: string;
+  download_path: string;
+  updated_at: string;
+}
+
+interface OpportunityDocumentsProps {
+  documents: OpportunityDocument[];
+}
+
+const OpportunityDocuments = ({ documents }: OpportunityDocumentsProps) => {
+  const t = useTranslations("OpportunityListing.documents");
+
+  const documentTable = (documents_filtered: OpportunityDocument[]) => {
+    return (
+      <>
+        <thead>
```

Comment:
Is there a reason why we shouldn't use the Trussworks table component here? https://trussworks.github.io/react-uswds/?path=/docs/components-table--docs

Reply:
I did use the trussworks table component, just this particular helper function returns only the table body which is identical to a regular html table, the <Table> component is further down in the code/return statement